spyderlib/plugins/configdialog.py

```diff
@@ -90,8 +91,20 @@ def apply_changes(self):
             self.save_to_conf()
             if self.apply_callback is not None:
                 self.apply_callback()
+
+            # Since the language cannot be retrieved by CONF and the language
+            # is needed before loading CONF, this is an extra method needed to
+            # ensure that when changes are applied, they are copied to a
+            # specific file storing the language value.
+            if getattr(self, 'save_lang', False):

```

Comment:
Cause this class is inherited by all config panes. But only the main will for sure have the save_lang method.